Gruntfile.js

```diff
@@ -2,11 +2,13 @@
 module.exports = function (grunt) {
 	grunt.initConfig({
 		xo: {
-			validate: ['test/fixture.js']
+			testTask: {
+				src: ['test/*.js']
+			}
```

Comment:
Oh, I'm so sorry. I didn't even think about that. I feel like a fucking idiot :tada: